src/flows/recommendation_api/corpus_candidate_sets/pockethits.py

```diff
@@ -26,7 +26,9 @@
     PUBLISHER as "PUBLISHER"
 FROM "ANALYTICS"."DBT"."SCHEDULED_CORPUS_ITEMS"
 WHERE SCHEDULED_SURFACE_ID = %(SURFACE_GUID)s
-AND SCHEDULED_CORPUS_ITEM_SCHEDULED_AT BETWEEN DATEADD(day, %(MAX_AGE_DAYS)s, CURRENT_DATE) AND CURRENT_DATE
+-- Pocket Hits content should be available on Home at 3am EST = 7am UTC. Override the scheduled time to 7am.
+-- BACK-1668: `scheduled_at` is always 12am UTC, which is unexpected because Pocket Hits goes out at 10am EST.
+AND DATEADD('hour', 7, DATE_TRUNC('DAY', SCHEDULED_CORPUS_ITEM_SCHEDULED_AT)) < CURRENT_TIMESTAMP
```

Comment:
i don't know how to make this better either, but we should follow up with G to see if he knows anything